Buildings/Experimental/DHC/CentralPlants/Cooling/Plant.mo

```diff
@@ -0,0 +1,383 @@
+within Buildings.Experimental.DHC.CentralPlants.Cooling;
+model Plant
+  "District cooling plant model"
+  extends Buildings.Experimental.DHC.CentralPlants.BaseClasses.PartialPlant(
+    have_eleCoo=true,
+    have_pum=true,
+    have_fan=true,
+    have_weaBus=true,
+    typ=Buildings.Experimental.DHC.Types.DistrictSystemType.Cooling);
+  // chiller parameters
+  parameter Integer numChi(
+    min=1,
+    max=2)=2
+    "Number of chillers, maximum is 2"
+    annotation (Dialog(group="Chiller"));
+  replaceable parameter Buildings.Fluid.Chillers.Data.ElectricEIR.Generic perChi
+    "Performance data of chiller"
+    annotation (Dialog(group="Chiller"),choicesAllMatching=true,
+   Placement(transformation(extent={{124,264},{138,278}})));
+  parameter Modelica.SIunits.MassFlowRate mCHW_flow_nominal
+    "Nominal chilled water mass flow rate"
+    annotation (Dialog(group="Chiller"));
+  parameter Modelica.SIunits.Pressure dpCHW_nominal
+    "Pressure difference at the chilled water side"
+    annotation (Dialog(group="Chiller"));
+  parameter Modelica.SIunits.HeatFlowRate QChi_nominal
```

Comment:
Optional: I think this could be capped with `max = 0` given that a negative value is expected.

Reply:
Sure, we have added “max=0” for QChi_nominal.